src/main/java/com/climatetree/demo/BackendApplication.java

```diff
@@ -0,0 +1,13 @@
+package com.climatetree.demo;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class BackendApplication {
```

Comment:
Should we name it ClimateTreeServer?

Reply:
https://github.com/climatetree/Backend-DTO/pull/1/commits/3d40c341ba9a13a18858d0d3c41843bb0df3a431